src/containers/Login.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import {   
+  Redirect 
+} from 'react-router-dom'
+import {AuthStore} from '../repos/UserRepo'
+
+
+export class Login extends React.Component {
```

Comment:
this is a simple fake login component.
1- user clicks a button
2- it invokes AuthStore.authenticate, on the call back it redirects to the caller (this prop is set in routes.js